### ISSUE TITLE
dotnet-runtime-2.1 on Ubuntu 18.04 installs 2.1.26 instead of 2.1.27

### ISSUE THREAD

**Issue by mikeharder** [dotnet/runtime#52331]
### Description
Running `apt-get install dotnet-runtime-2.1` on Ubuntu 18.04 installs 2.1.26 instead of 2.1.27.

### Repro Steps
```
> docker run -it --rm ubuntu:18.04

# apt install wget

# wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
# dpkg -i packages-microsoft-prod.deb

# apt-get update
# apt-get install -y apt-transport-https
# apt-get update
# apt-get install -y dotnet-runtime-2.1

Selecting previously unselected package dotnet-runtime-2.1.
Preparing to unpack .../12-dotnet-runtime-2.1_2.1.26-1_amd64.deb ...
Unpacking dotnet-runtime-2.1 (2.1.26-1) ...

# dotnet --info

Host (useful for support):
  Version: 2.1.26
  Commit:  c5668293c2

.NET Core SDKs installed:
  No SDKs were found.

.NET Core runtimes installed:
  Microsoft.NETCore.App 2.1.26 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by NikolaMilosavljevic** [comment]
[Triage] @rbhanda we don't see 2.1.27 at https://packages.microsoft.com/ubuntu/18.04/prod/pool/main/d/dotnet-runtime-2.1/ Was it released for Ubuntu 18.04?

**Comment by NikolaMilosavljevic** [comment]
[Triage] Does not repro anymore - 2.1.27 and 2.1.28 are available.
